chainntnfs/txconfnotifier_test.go

```diff
@@ -124,7 +124,7 @@ func TestTxConfFutureDispatch(t *testing.T) {
 		NumConfirmations: tx1NumConfs,
 		Event:            chainntnfs.NewConfirmationEvent(tx1NumConfs),
 	}
-	if _, _, err := tcn.Register(&ntfn1); err != nil {
+	if _, err := tcn.Register(&ntfn1); err != nil {
```

Comment:
looks like also can just be squashed with the prev commit changing this

Reply:
fixed